commands/import.go

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+)
+
+const fakeUserID = "nouser"
+
+var ImportCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Management of imports",
+}
+
+var ImportUploadCmd = &cobra.Command{
+	Use:     "upload [filepath]",
+	Short:   "Upload import files",
+	Example: " import upload import_file.zip",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(importUploadCmdF),
+}
+
+var ImportListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List import files",
```

Comment:
Would it make sense to use `ls` here?
```suggestion
	Use:     "list",
	Aliases: []string{"ls"},
	Short:   "List import files",
```

Reply:
Less typing sounds good to me.